src/main/java/com/sumologic/client/Credentials.java

```diff
@@ -7,37 +7,62 @@
  * @author Sebastian Mies
  */
 public class Credentials {
-    private String email;
-    private String password;
+    private String accessId;
+    private String accessKey;
 
     /**
-     * Constructs the credentials using email and password.
+     * Constructs the credentials using accessId and accessKey.
      *
-     * @param email Your email
-     * @param password Your password
+     * @param accessId Your access id
+     * @param accessKey Your access key
      */
-    public Credentials(String email, String password) {
-        this.email = email;
-        this.password = password;
+    public Credentials(String accessId, String accessKey) {
```

Comment:
I strongly disagree with this. If we are making backend changes that will break 1/2 the users of this class, and we didn't design it so that it's possible to tell who will be broken, then IMO the responsible thing to do is deprecate the class and provide something new for everyone to use.